pkg/runtest/run.go

```diff
@@ -553,27 +554,20 @@ func checkCallResult(req *runRequest, isC bool, run, call int, info *flatrpc.Pro
 		return nil
 	}
 	if req.ExecOpts.EnvFlags&flatrpc.ExecEnvSignal != 0 {
+		callName := req.Prog.Calls[call].Meta.CallName
+		// Pseudo-syscalls that might not provide any coverage when invoked.
+		noCovSyscalls := []string{"syz_btf_id_by_name", "syz_kvm_assert_syzos_uexit"}
+		isNoCov := slices.Contains(noCovSyscalls, callName)
```

Comment:
The proposal is to stop changing 'calls' in this function. The caller side will modify it if `checkCallResult` returned no error.
It allows to simplify `isNoCov` usage here.

Reply:
Will do in a follow-up patch.